src/runner/PlutoRunner.jl

```diff
@@ -2203,55 +2228,81 @@ const original_stderr = stderr
 
 const old_logger = Ref{Union{Logging.AbstractLogger,Nothing}}(nothing)
 
-struct PlutoLogger <: Logging.AbstractLogger
+struct PlutoCellLogger <: Logging.AbstractLogger
     stream # some packages expect this field to exist...
     log_channel::Channel{Any}
-    currently_running_cell_id::Ref{UUID}
+    cell_id::UUID
+    workspace_count::Int # Used to invalidate previous logs
+end
+function PlutoCellLogger(notebook_id, cell_id)
+    notebook_log_channel = pluto_log_channels[notebook_id]
+    PlutoCellLogger(nothing, notebook_log_channel, cell_id, moduleworkspace_count[])
+end
+
+struct CaptureLogger <: Logging.AbstractLogger
+    stream
+    logger::PlutoCellLogger
+    logs::Vector{Any}
+end
+
+Logging.shouldlog(cl::CaptureLogger, args...) = Logging.shouldlog(cl.logger, args...)
+Logging.min_enabled_level(::CaptureLogger) = min(Logging.Debug, stdout_log_level)
+Logging.catch_exceptions(::CaptureLogger) = Logging.catch_exceptions(cl.logger)
+function Logging.handle_message(pl::CaptureLogger, level, msg, _module, group, id, file, line; kwargs...)
+    Logging.handle_message(pl.logger, level, msg, _module, group, id, file, line; kwargs...)
```

Comment:
what is this line for?

Reply:
It looks like it creates this behaviour. It happens very fast, so I made a video and screenshotted three consecutive frames:

<img width="595" alt="Scherm­afbeelding 2022-11-09 om 13 20 45" src="https://user-images.githubusercontent.com/6933510/200829149-2957bb86-c772-4c99-aa6b-a4abdf8c0f55.png">

<img width="595" alt="Scherm­afbeelding 2022-11-09 om 13 20 59" src="https://user-images.githubusercontent.com/6933510/200829177-5074ae89-6735-44e3-a20f-ab263117c664.png">

<img width="595" alt="Scherm­afbeelding 2022-11-09 om 13 21 06" src="https://user-images.githubusercontent.com/6933510/200829198-89950dd8-b457-47b7-a12b-a5ecb59e6f71.png">